### ISSUE TITLE
VoiceOver issues

### ISSUE THREAD

**Issue by phillipthelen** [HabitRPG/habitrpg-ios#737]
Mounts aren’t read by VoiceOver

I have reported this before, but mounts are unfortunately still not accessible by VoiceOver. Everything works well with picking and choosing pets, but when it comes to mounts, VoiceOver acts as if there was a blank page.
